kubernetes/annotations_and_labels.md

```diff
@@ -36,6 +39,10 @@ This page defines common annotations and labels we set in Kubernetes objects.
   services (i.e. K8s components) or `managed` for Giant Swarm managed services
   (i.e. networking, DNS, monitoring, ingress controller, etc.). Latter would be
   managed by `chart-operator`.
+- `machine-pool.giantswarm.io/subnet`: value should contain the name of the
```

Comment:
I think this is not needed. The subnet can be called like the node pool it was created for. Who can we talk to so we can discuss if it could be changed in AWS?

Reply:
My thinking was that it is easier that we add it for Azure now, as we are in development, than to change it for AWS which is already working this way for some time.

But I guess we could skip using the label for subnet anyway, if we are to follow.

P.S. We could maybe check out if we can have node pool subnet naming compatible with CAPZ.

Reply:
Subnet annotation is removed now.